### PR TITLE
set the focus on title when creating or editing a post

### PR DESCRIPTION
**Pull request by yceruto**
Perhaps it would be good to suggest to the user where to begin writing the post.

### CODE REVIEW

File: src/AppBundle/Form/PostType.php [hunk 1]
```diff
@@ -39,7 +39,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         //     $builder->add('title', null, array('required' => false, ...));
 
         $builder
-            ->add('title', null, array('label' => 'label.title'))
+            ->add('title', null, array(
+                'attr' => array('autofocus' => true),
```
Comment (rasanga) [inline src/AppBundle/Form/PostType.php]: I prefer to keep attributes on the form view, but I would like to knows others intentions

Comment (voronkovich) [inline src/AppBundle/Form/PostType.php]: @rasanga, I agree with you. Keeping attributes on the template is more flexible approach than this. But this PR just follows the existing code style. I think you could refactor the form templates and create a PR.

Comment (rasanga) [inline src/AppBundle/Form/PostType.php]: @voronkovich good thinking let me create a PR with form attributes on the template

Comment (yceruto) [inline src/AppBundle/Form/PostType.php]: @rasanga, I agree too, but as @voronkovich says, just follows the existing code style. https://github.com/symfony/symfony-demo/blob/master/src/AppBundle/Form/PostType.php#L45

Another way is using javascripts `$('#app_post_title').focus();` within edit and new templates, but I still prefer this PR. Is simple and without dependencies.

Comment (rasanga) [inline src/AppBundle/Form/PostType.php]: @yceruto I didn't know we were using the 'attr' tag on the form type.

**Comment by javiereguiluz** [comment]
@yceruto thanks for helping us improve the demo application!

**Comment by yceruto** [comment]
:+1:

**Pull request merged**